spec.html

```diff
@@ -1003,8 +1003,8 @@ <h1>Static Semantics</h1>
       <h1>Mathematical Operations</h1>
       <p>This specification makes reference to these kinds of numeric values:</p>
       <ul>
-        <li><em>Mathematical values</em>: Arbitrary real numbers, used as the default numeric type.</li>
-        <li><em>Extended mathematical values</em>: Mathematical values together with +&infin; and -&infin;.</li>
+        <li><dfn id="mathematical-value" variants="mathematical value,mathematical values">Mathematical values</dfn>: Arbitrary real numbers, used as the default numeric type.</li>
+        <li><dfn id="extended-mathematical-value" variants="extended mathematical value,extended mathematical values">Extended mathematical values</dfn>: Mathematical values together with +&infin; and -&infin;.</li>
```

Comment:
This is beginning to fix #2786, :+1:

Reply:
I made this change so that everything mentioned in the new section links somewhere useful, but I should probably split it into a separate PR since it's useful on its own.